### PR TITLE
cdc: add more detail about the authorization

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by flowbehappy — APPROVED** [review]

File: ticdc/ticdc-sink-to-kafka.md [hunk 1]
```diff
@@ -159,6 +159,15 @@ URI 中可配置的的参数如下：
...
     - 对 Cluster 资源类型的 `DescribeConfigs` 权限。
 
+| Resource Type | Operation      | Usage                            |
+| :-------------| :------------- | :--------------------------------|
+| Cluster       | DescribeConfig | changefeed 运行过程中，获取集群元数据 |
+| Topic         | Describe       | changefeed 启动时，尝试创建 Topic   |                
+| Topic         | Create         | changefeed 启动时，尝试创建 Topic   |
+| Topic         | Write          | 发送数据到 Topic                   | 
+
+* 创建或启动 changefeed 时，如果指定的 Kafka Topic 已存在，可以不用开启 `Describe` 和 `Create` 权限。
```
Comment (hfxsd) [inline ticdc/ticdc-sink-to-kafka.md:169]:
```suggestion
  各权限的使用场景如下：
  
    | 资源类型 | 操作类型      |  使用场景                            |
    | :-------------| :------------- | :--------------------------------|
    | Cluster      | `DescribeConfig` | Changefeed 运行过程中，获取集群元数据 |
    | Topic         | `Describe`           | Changefeed 启动时，尝试创建 Topic   |                
    | Topic         | `Create`              | Changefeed 启动时，尝试创建 Topic   |
    | Topic         | `Write`                | 发送数据到 Topic                   | 

    创建或启动 changefeed 时，如果指定的 Kafka Topic 已存在，可以不用开启 `Describe` 和 `Create` 权限。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 2]
```diff
@@ -159,6 +159,17 @@ URI 中可配置的的参数如下：
     - 对 Topic [资源类型](https://docs.confluent.io/platform/current/kafka/authorization.html#resources)的 `Create` 、`Write` 和 `Describe` 权限。
     - 对 Cluster 资源类型的 `DescribeConfigs` 权限。
 
+  各权限的使用场景如下：
+  
+    | 资源类型 | 操作类型      |  使用场景                            |
+    | :-------------| :------------- | :--------------------------------|
+    | Cluster      | `DescribeConfig` | Changefeed 运行过程中，获取集群元数据 |
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:166]: L160 行里是  `DescribeConfigs`，多了个 s，不确定哪个是对的

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: ticdc/ticdc-sink-to-kafka.md [hunk 3]
```diff
@@ -159,6 +159,17 @@ URI 中可配置的的参数如下：
     - 对 Topic [资源类型](https://docs.confluent.io/platform/current/kafka/authorization.html#resources)的 `Create` 、`Write` 和 `Describe` 权限。
     - 对 Cluster 资源类型的 `DescribeConfigs` 权限。
 
+  各权限的使用场景如下：
+  
+    | 资源类型 | 操作类型      |  使用场景                            |
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:164]: 第二列的名字要不要写成 权限

Comment (3AceShowHand) [reply ticdc/ticdc-sink-to-kafka.md:164]: 不用，就是类型。

File: ticdc/ticdc-sink-to-kafka.md (hunk 2 above)
Comment (3AceShowHand) [reply ticdc/ticdc-sink-to-kafka.md:166]: DescribeConfig

**Review by hfxsd — APPROVED** [review]

**Pull request merged**